src/Cassandra/Exceptions/NoHostAvailableException.cs

```diff
@@ -18,6 +18,7 @@
 using System.Collections.Generic;
 using System.Net;
 using System.Runtime.Serialization;
```

Comment:
Remove unused import

Reply:
Will do, thanks!

Reply:
ahh, I've just checked: `System.Runtime.Serialization` is unused for .NET Core target but needed for building against .NET Framework (see `SerializableAttribute` below): I'll surround it with `#if !NETCORE` directive to make it more clear.

Reply:
Good solve!